packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -586,11 +607,11 @@ export const useTransactionHistory = (
 
   return {
     data: {
-      transactions,
+      transactions: dedupedTransactions,
       numberOfDays: oldestTransactionDaysAgo
     },
     loading: fetching,
-    completed: transactions.length === data.length,
+    completed: dedupedTransactions.length >= data.length,
```

Comment:
Let's throw if deduped.length > data.length, and return to the previous implementation

Reply:
I moved deduping to getting txs without statuses, I don't think we need to throw anymore